### ISSUE TITLE
Consider inputs

### ISSUE THREAD

**Issue by cobraz** [indivorg/theme#49]
For quite some time now, we've used a special input component in the application.

We should consider moving to a standardized input component. Before doing so, we should ensure we've got a design for the forms. I can help out with that this weekend or in my free time.

**Comment by braaar** [comment]
Agreed. The current input fields are actually a bit confusing since the border around them is quite a bit bigger than the actual input field, so you can end up having to click around inside the box to find the actual input.


https://user-images.githubusercontent.com/5765650/143435082-c25c1841-e6d9-47b7-b0c6-9fa3d3c55f6b.mov
